source/tutorials/test-strategy/test-strategy.rst

```diff
@@ -54,41 +54,20 @@ Android Validation Out-Scope
 * Pre-Certification
 * Documentation
 
-Validation Activates Key Flow
+Validation Activity Flow
 -----------------------------
 
 .. figure:: images/validation-activates-key-flow.png
```

Comment:
@EdieM you already renamed the image filename, it must be reflected in the above link.

Reply:
Besides the missing image file warning, LGTM.